internal/service/user.go

```diff
@@ -35,10 +36,11 @@ func (service *User) CreateUser(ctx context.Context, signup request.Signup, pass
 	}
 
 	user := domain.User{
-		ID:       userid,
-		Email:    signup.Email,
-		Role:     signup.Role,
-		Password: string(encryptedPassword),
+		ID:                   userid,
+		Email:                signup.Email,
+		Role:                 signup.Role,
+		Password:             string(encryptedPassword),
+		RequirePasswordReset: true,
```

Comment:
You have this field default as false. There is no need to have your password changed right after you signed up. 
We should conditionally have this field updated.

Reply:
what does mean:

> We should conditionally have this field updated.

Reply:
> We should conditionally have this field updated.

hat do you mean by this, what is conditionally for you

Reply:
What I meant is that it's fine with default value. And we should have this field updated on reaching some conditions. E.g. having scheduler that checks that passwords are changed once in a 3 months, password was leaked, etc.